cmd/jimmsrv/main.go

```diff
@@ -141,19 +142,26 @@ func start(ctx context.Context, s *service.Service) error {
 		return errors.E("jimm session store secret must be at least 64 characters")
 	}
 
+	hostKeyRaw := os.Getenv("JIMM_SSH_HOST_KEY")
+	fingerprints, err := ssh.GetFingerprintsFromPrivateKey([]byte(hostKeyRaw))
+	if err != nil {
+		return errors.E("Cannot parse hostkey")
```

Comment:
could we please start with a lower case letter? i know, we're consistently inconsistent, but still..